test/e2e/play_kube_test.go

```diff
@@ -1693,7 +1693,7 @@ func createAndTestSecret(podmanTest *PodmanTestIntegration, secretYamlString, se
 	secretListQuiet := podmanTest.Podman([]string{"secret", "list", "--quiet"})
 	secretListQuiet.WaitWithDefaultTimeout()
 	Expect(secretListQuiet).Should(Exit(0))
-	Expect(kube.OutputToString()).Should(ContainSubstring(secretListQuiet.OutputToString()))
+	Expect(secretListQuiet.OutputToString()).Should(ContainSubstring(secretListQuiet.OutputToString()))
```

Comment:
What is broken?  Given the test passes in CI, it seems like some change in this PR caused the test to fail.

Reply:
fixed